pkg/api/api.go

```diff
@@ -11,32 +11,52 @@ import (
 	"strings"
 	"time"
 
+	"nodemon/pkg/entities"
+	"nodemon/pkg/storing/events"
+	nodesStor "nodemon/pkg/storing/nodes"
```

Comment:
I don't get it. Why rename `nodes` to `nodesStor` here? `nodes.Storage` even sounds better than `nodesStor.Storage`.